CHANGELOG.md

```diff
@@ -104,6 +104,7 @@ Sjekk ut [release notes](./releasenotes/1.8.0.md) for høydepunkter og mer detal
 
 ### Feilrettinger
 
+- Rettet en feil hvor Excel eksporten gir feilmelding når datofelt mangler data. [#1180](https://github.com/Puzzlepart/prosjektportalen365/issues/1180)
```

Comment:
denne tror jeg ligger en helt annen vei, husk å legge den i bunnen på feilrettinger for 1.8.3

Reply:
Den ligger under 1.8.0